README.md

```diff
@@ -130,6 +130,29 @@ The following steps describe how to create the service principal, assign the rol
 
 NOTE: to manage service principals created with `az ad sp create-for-rbac`, visit the [Azure portal](https://portal.azure.com), navigate to your Azure Active Directory, then select **Manage** > **App registrations** on the left-hand menu. Your service principal should appear in the list. Select a principal to navigate to its properties. You can also manage role assignments using the [az role assignment](https://docs.microsoft.com/cli/azure/role/assignment?view=azure-cli-latest) command.
```

Comment:
We should also add a mention about this new input in the introduction section : **GitHub Action for Azure Login**
With the Azure login Action, you can automate your workflow to do an Azure login using Azure service principal and run Azure CLI and Azure PowerShell scripts.

By default, the action only logs in with the Azure CLI (using the az login command). To log in with the Az PowerShell module, set enable-AzPSSession to true. To login to Azure tenants without any subscriptions,  set the optional parameter `allow-no-subscriptions` to `true`.

This repository contains GitHub Action for Azure Login.